modules/ding_ipe_filter/css/ipe.css

```diff
@@ -32,6 +33,6 @@ div.ipe-popup .item-list ul li {
   overflow: visible;
 }
 .ding-ipe-menu-item {
-  width: calc(100% - 18px);
```

Comment:
As far as I can tell changes to `ipe.css` do not seem related to the change description or issue. Are they intentional?

Reply:
They solve the problems described in the issue. However not reflected in the commit message.

Reply:
OK. Thanks for the update.